geomesa-tools/src/test/scala/org/locationtech/geomesa/tools/ScaldingDelimitedIngestJobTest.scala

```diff
@@ -277,6 +288,50 @@ class ScaldingDelimitedIngestJobTest extends Specification{
       f.getAttribute(5) must beAnInstanceOf[Geometry]
     }
 
+    "throw an exception when lat/lon field names are provided in args but aren't in sft" in {
```

Comment:
preference? some code of course will print them out in lexicographic order though i prefer lon/lat since that is what we always end up using, right?